client/pages/lobby.jsx

```diff
@@ -68,15 +70,25 @@ export default function Lobby(props) {
     const req = {
       method: 'POST'
     };
-    fetch('/api/game', req);
+    fetch('/api/game', req)
+      .then(res => res.json())
+      .then(result => {
+        const { gameId } = result;
+        window.location.href = `/fight?gameId=${gameId}`;
```

Comment:
Since we're using React Router, can use this super convenient useHistory hook that comes with it

https://reactrouter.com/web/api/Hooks/usehistory